recipes/dynare_preprocessor/run_test.sh

```diff
@@ -0,0 +1,2 @@
+$PREFIX/bin/dynare-preprocessor | grep -q "Missing model file!"
+$PREFIX/bin/dynare-preprocessor $RECIPE_DIR/example1.mod | grep -q "Preprocessing completed."
```

Comment:
```suggestion
dynare-preprocessor | grep -q "Missing model file!"
dynare-preprocessor $RECIPE_DIR/example1.mod | grep -q "Preprocessing completed."
```
You shouldn't need to list the full path of the installed binary. If it's not on the path in the test environment, then you package is broken!